doc/user-guide/client-certificate.md

```diff
@@ -44,7 +44,7 @@ For the details please refer to [XEP-0178 Best Practices for Use of SASL EXTERNA
 You need to enable one of the following authentication backends by using the [`auth_method` option](../Advanced-configuration.md#authentication) in the MongooseIM configuration file.
 
 * `pki` - accepts user credentials,
-* `http` - accepts user credentials if the provided certificate is [known and valid](../authentication-backends/HTTP-authentication-module.md#method-get_certs)
+* `http` - accepts user credentials if the provided certificate is [known and valid](../authentication-backends/http#method-get_certs)
```

Comment:
```suggestion
* `http` - accepts user credentials if the provided certificate is [known and valid](../../authentication-methods/http#method-get_certs)
```

Reply:
I did nothing to this file yet (it's a separate story), but I will fix this.

Reply:
I thought that you changed the `HTTP-authentication-module` to `http`. Anyway, thanks for changing that.